great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py

```diff
@@ -82,7 +82,6 @@ class ExpectMulticolumnSumToEqual(MulticolumnMapExpectation):
 
     @classmethod
     @renderer(renderer_type=LegacyRendererType.PRESCRIPTIVE)
-    @renderer(renderer_type="renderer.prescriptive")
```

Comment:
Let's remove string usage and any duplicate decorators

Reply:
I dig getting rid of magic strings and duplicate renderers, but I have to imagine we should be moving away from `LegacyRendererType`, right?

Reply:
Yup! This has no functional change just cleaning up a few inconsistencies